packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/selectors.ts

```diff
@@ -109,7 +111,7 @@ export const getData = (state, coin, isCoinErc20) =>
       selectors.core.settings.getCurrency,
       () => selectors.core.walletOptions.getCoinModel(state, coin)
     ],
-    (userSearch, pagesR, currencyR, coinModelR) => {
+    (userSearch, pagesR, currencyR, coinModelR, supportedCoinsR) => {
```

Comment:
surprised this compiled, you dont have a matching selector for this 5th argument.  move the selector from line 138 to the array of selectors that starts on line 108